packages/@react-spectrum/tooltip/docs/Tooltip.mdx

```diff
@@ -53,7 +53,7 @@ keywords: [tooltip]
 
 ## Content
 
-Tooltips accept content as children. All content should be internationalized.
+The TooltipTrigger accepts exactly two children: the Tooltip and the element which triggers the display of the Tooltip. The trigger element must be the first child passed into the TooltipTrigger. All content should be internationalized.
```

Comment:
```suggestion
The TooltipTrigger accepts exactly two children: the element which triggers the display of the Tooltip and the Tooltip itself. The trigger element must be the first child passed into the TooltipTrigger. All content should be internationalized.
```

Just want to be consistent in the order that we talk about them so it naturally leads the user to put them in the right order. You'll notice the DialogTrigger sentence does the same

Reply:
Good catch. I'll correct this in MenuTrigger too.